backend/prisma/schema.prisma

```diff
@@ -66,16 +66,11 @@ model User {
 
   email       String @unique
   password    String @map("password")
-  yearOfStudy Int    @map("year_of_study")
-
-  otp        String? @default("")
-  jwt        String? @default("")
-  isVerified Boolean @default(false) @map("is_verified")
 
   profilePic String? @unique @map("profile_pic")
 
   institution   Institution @relation(fields: [institutionId], references: [id], onDelete: Cascade)
-  institutionId String      @map("institution_id")
+  institutionId String
```

Comment:
why are you getting rid of the map

Reply:
not sure why I did that, putting it back